### PR TITLE
[DynaConf] added virtwho yaml template file

### PR DESCRIPTION
**Pull request by ColeHiggins2**
Dynaconf conversion virtwho template file

### CODE REVIEW

File: conf/virtwho.yaml.template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# The hypervisor type, one of the values: esx, xen, hyperv, rhevm, libvirt, kubevirt
+  HYPERVISOR_TYPE:
+  # The hostname or ip or other values for hypervisor server, depends on the type
+  HYPERVISOR_SERVER:
+  # Admin username to access hypervisor server
+  HYPERVISOR_USERNAME:
+  # Admin password to access hypervisor server
+  HYPERVISOR_PASSWORD:
+  # The config file to connect hypervisor, only for kubevirt
+  #hypervisor_config_file=
```
Comment (JacobCallahan) [inline conf/virtwho.yaml.template:11]: caps this for consistency

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

**Review by rplevka — APPROVED** [review]

File: conf/virtwho.yaml.template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# The hypervisor type, one of the values: esx, xen, hyperv, rhevm, libvirt, kubevirt
+  HYPERVISOR_TYPE:
+  # The hostname or ip or other values for hypervisor server, depends on the type
+  HYPERVISOR_SERVER:
+  # Admin username to access hypervisor server
+  HYPERVISOR_USERNAME:
+  # Admin password to access hypervisor server
+  HYPERVISOR_PASSWORD:
+  # The config file to connect hypervisor, only for kubevirt
+  #HYPERVISOR_CONFIG_FILE=
```
Comment (JacobCallahan) [inline conf/virtwho.yaml.template:11]: Ahhh I missed this the first time! 
```suggestion
  #HYPERVISOR_CONFIG_FILE:
```

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

File: conf/virtwho.yaml.template (hunk 2 above)
Comment (ColeHiggins2) [reply conf/virtwho.yaml.template:11]: this was comment out originally, it is not used. but I can change it

File: conf/virtwho.yaml.template [hunk 3]
```diff
@@ -0,0 +1,23 @@
+VIRTWHO:
```
Comment (mshriver) [inline conf/virtwho.yaml.template:1]: @ColeHiggins2 and I discussed the scope of this file yesterday, and this PR will need some significant changes.

virtwho.properties is what is actually utilized by the virtwho tests, the VirtwhoSettings class that is defined, and that has this structure in robotteo.properties, is not in use by any of the tests or framework.

This file should have a structure where there are hypervisor details under a map for each hypervisor type, like:

```
VIRTWHO:
  ESX:
    SERVER:
    USERNAME:
    GUEST:
    ETC:
  XEN:
    SERVER:
    USERNAME:
    GUEST:
    ETC:
```

**Pull request closed without merging**